src/buildMathML.js

```diff
@@ -515,6 +515,14 @@ groupTypes.mclass = function(group, options) {
     return new mathMLTree.MathNode("mstyle", inner);
 };
 
+groupTypes.not = function(group, options) {
+    const solidus = new mathMLTree.TextNode('&#x29F8;');
```

Comment:
Why an XML escape sequence instead of a JavaScript one? I disabled the CSS rules which ide our MathML, and sure enough, Firefox renders the escape sequence as is, instead of the symbol you meant.